Makefile

```diff
@@ -22,7 +22,7 @@ eget:
 	go build -trimpath -ldflags "-s -w $(GOVARS)" .
 
 test: eget
-	cd test; EGET_BIN= TEST_EGET=../eget go run test_eget.go
```

Comment:
When running `make test` it will successfully try to read the config file from `~/.config` and then if for instance you have the global `target` set it will use that rather than the test directory. I had generated a blank `eget.toml` in the test directory which successfully parses and then skips the rest of the config lookups. But you could also just have a `--config-file`/`--no-config` flag that does the same thing of skipping config parsing. 

I realize now though that I didn't update the  `.gitignore` to accommodate the blank config file in `./test` if this approach is kept.

Reply:
Okay I see -- this is fine, just make sure to commit the empty toml file.